setup/setup_rirs.py

```
@@ -0,0 +1,406 @@
+            "source_directory": str(source_path),
+            "installation_date": str(Path().cwd()),
+            "file_count": copied_count,
+            "total_size_mb": sum(f.stat().st_size for f in self.rir_data_dir.rglob("*") if f.is_file()) / (1024 * 1024),
+            "compatible_with": ["wakeword_training", "speech_enhancement", "voice_activity_detection"]
+        }
+
+        metadata_file = self.metadata_dir / "custom_rirs_metadata.json"
+        with open(metadata_file, 'w') as f:
+            json.dump(metadata, f, indent=2)
+
+        print(f"✅ Custom RIR dataset created successfully!")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)